.idea/copyright/Apache2.xml

```diff
@@ -1,6 +0,0 @@
-<component name="CopyrightManager">
```

Comment:
This file should be included in an ideaconfig folder so that you only have to copy it over to .idea folder after import.
It is so useful and a waste of time if everybody has to configure this on his own.